programmers-coding-test/soongu/sort/위에서아래로.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+let input = fs.readFileSync("/dev/stdin").toString().split('\n');
+
+const N = input[0];
+input = input.slice(1);
+const array = Array.from(Array(N).fill(0));
+for (let i = 0; i < N; i++) {
+    array[i] = input[i];
+}
+
+array.sort((a, b) => b - a);
+for (let i = 0; i < N; i++) {
+    process.stdout.write(` ${array[i]} `);
+}
```

Comment:
앞 공백은 없어도 됩니다! 어차피 뒷 공백이 있어서 띄어쓰기 된 채로 출력돼요!
```suggestion
for (let i = 0; i < N; i++) {
    process.stdout.write(`${array[i]} `);
}
```

Reply:
이렇게 할 수도 있습니다(이미 알고계시겠지만!ㅎㅎ)
```suggestion
array.forEach(el => process.stdout.write(`${el} `);
```